### PR TITLE
Adds new line item tax fields for upcoming API release

### PR DESCRIPTION
**Pull request by ryankazokas**
* Adds new fields with unit tests.
* Will need to add e2e tests when API becomes available.

### CODE REVIEW

**Comment by ryankazokas** [comment]
@mb-ca @mvx24 @ShannonTalarico this should provide the contract you guys need to start working with these fields. The only thing missing is the API e2e tests which we can add once we release the new API versions.

File: test/orders/service/OrdersService.spec.ts [hunk 1]
```diff
@@ -602,10 +626,18 @@ Code: 15 Message: Requested business cannot be found.`;
...
+        expect(actualOrder.lineItems[0].taxes![0].price).to.equal(9.99);
+        expect(actualOrder.lineItems[0].taxes![0].rate).to.equal(0.06);
+        expect(actualOrder.lineItems[0].taxes![1].title).to.equal('NV State Tax');
+        expect(actualOrder.lineItems[0].taxes![1].price).to.equal(1.00);
+        expect(actualOrder.lineItems[0].taxes![1].rate).to.be.undefined;
       });
     });
 
     it('And valid OrderCreateRequest with an actionId when creating an order, Then return created order', () => {
+      console.log(JSON.stringify(singleOrder));
```
Comment (mvx24) [inline test/orders/service/OrdersService.spec.ts:640]: @ryankazokas Forgot a debugging console.log?

**Comment by mvx24** [comment]
@ryankazokas Thanks! I'm not super familiar with the testing in here but it looks good to me.

File: test/orders/service/OrdersService.spec.ts (hunk 1 above)
Comment (ryankazokas) [reply test/orders/service/OrdersService.spec.ts:640]: Yes, thank you. Good catch.

**Review by mb-ca — APPROVED** [review]

**Pull request merged**